### PR TITLE
docs: add guide to integrate with 3rd party UI libraries

### PR DESCRIPTION
**Pull request by devCrossNet**
related to #220

### What is accomplished by your PR?
This PR will add a guide to integrate vuetify

### Is there something controversial in your PR?
nope

### Link to the Issue
#220 

# Checklist

### New Feature / Bug Fix

- [x] Run unit tests to ensure all existing tests are still passing
- [ ] Add new passing unit tests to cover the code introduced by your PR
- [x] Add new documentation for the code introduced by your PR

### CODE REVIEW

**Comment by glederer** [comment]
Wow.. that was fast. Makes even consider using vuetify. @devCrossNet  I am sory I am no expert enought to actually the quality of the config part, but I did read all our guides these past 2 days, just read the diff of this addendum and this made me have a suggestion ! in the tool section, creation a module actually create "not so empty" modules with pages actually using counter components... not relevant if one actually  start using this new vuetify approach.. what do you think ?

**Comment by devCrossNet** [comment]
You can change the templates for the cli by yourself on a project base. This is part of the philosophy of the VueStarter. You can change everything that it fits your projects needs.

File: docs/guide/third-party.md [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+  ],
+});
+
+module.exports = isomorphicConfig;
+
+```
+
+### Register vuetify
+
+I order to use the components you have to register vuetify in `./src/app.ts`
```
Comment (xMartin) [inline docs/guide/third-party.md]: Typo: "In order..."

**Comment by Heimathandel** [comment]
Thank you for the quick merge!

**Comment by devCrossNet** [comment]
good point @xMartin will change it

**Pull request merged**